src/pages/MerchantGiftCards/MerchantGiftCards.tsx

```diff
@@ -0,0 +1,144 @@
+import React, { useState, useEffect } from 'react';
+import { useHistory } from 'react-router-dom';
+import {
+  getMerchantGiftCards,
+  getSeller,
+} from '../../utilities/api/interactionManager';
+import Loader from '../../components/Loader/Loader';
+import styles from './styles.module.scss';
+import ErrorPage from '../../components/404Page';
+const FilterableTable = require('react-filterable-table');
```

Comment:
Is this something we can lazy import? Seems like a large library

Reply:
Ahh nvm I see you lazy import in App.tsx :).

Reply:
Oh yeah, don't want to load this all in the main app 😎